skipruntime-ts/native/src/Extern.sk

```diff
@@ -607,7 +607,9 @@ fun getAllOfRuntime(
     }
   | _ ->
     SKStore.runWithResult(context ~> {
-      getAll(context, resource, params(jsonParams), optRequest)
+      res = getAll(context, resource, params(jsonParams), optRequest);
+      context.update();
```

Comment:
Why is this needed now and not before?

Reply:
I realized that this was missing since the status management, it works without but it would be cleaner with. But it's true that it's not part of this commit.